make/tasks/homebrew/release-formulas.py

```diff
@@ -25,8 +25,7 @@
 
 def _set_logging():
     LOG_LEVEL = os.environ.get("LOG_LEVEL", "INFO")
-    print("LOG_LEVEL = {}".format(LOG_LEVEL))
-
+    print(f"LOG_LEVEL = {LOG_LEVEL}")
```

Comment:
This change should be uniform for other cases where the `format` method is used.